.ci/deps.sh

```diff
@@ -101,6 +101,7 @@ go get -u golang.org/x/tools/cmd/goimports
 go get -u sourcegraph.com/sqs/goreturns
 go get -u github.com/jayvdb/gotype
 go get -u github.com/kisielk/errcheck
+go get -u github.com/shurcooL/markdownfmt
```

Comment:
unrelated change, should go into a separate commit. If it fixes the test, split it out to a commit before the feature commit